tests/normalizers/keywordProtector.test.js

```diff
@@ -0,0 +1,41 @@
+import KeywordProtector from '../../src/normalizers/keywordProtector';
+
+describe('Keyword Protector tests', () => {
+
+    const kp = new KeywordProtector([ 'Hopper', 'Price Freeze' ], {Hopper: {ja: 'ホッペー', ko: '홉 따는 사람'}}); 
+
+    test('keyword with mapped values', async() => {
+
+        expect(kp.decoder([ { t: "s", v: "Hopper: Flights, Hotels & Cars"} ]))
+            .toMatchObject([ { t: "x", v: "protector:Hopper"}, { t: "s",  v: ": Flights, Hotels & Cars" } ]);
+
+        expect(kp.defaultEncoder("protector:Hopper", { targetLang: 'ja', prj: 'l10n-adhoc-requests' }))
+            .toMatch(/Hopper/);
```

Comment:
i thought encoders return strings. does that really return a regex?

Reply:
You're right.  Strings is better.  Changed